components/helptext/index.css

```diff
@@ -112,36 +92,63 @@
 	.spectrum-HelpText-text {
 		padding-block-start: var(--mod-helptext-top-to-text, var(--spectrum-helptext-top-to-text));
 		padding-block-end: var(--mod-helptext-bottom-to-text, var(--spectrum-helptext-bottom-to-text));
-		line-height: var(--mod-helptext-line-height, var(--spectrum-line-height-100));
+		line-height: var(--mod-helptext-line-height, var(--spectrum-helptext-line-height));
+		font-family: var(--mod-helptext-font-family, var(--spectrum-helptext-font-family));
+		font-style: var(--mod-helptext-font-style, var(--spectrum-helptext-font-style));
+		font-weight: var(--mod-helptext-font-weight, var(--spectrum-helptext-font-weight));
 	}
+}
 
-	&.spectrum-HelpText--neutral {
-		.spectrum-HelpText-text {
-			color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
-		}
+/* Variants -- Neutral, negative, disabled */
+.spectrum-HelpText--neutral {
+	--spectrum-helptext-content-color-default: var(--spectrum-neutral-subdued-content-color-default);
+	--spectrum-helptext-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
 
-		.spectrum-HelpText-validationIcon {
-			color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
-		}
+	.spectrum-HelpText-text {
+		color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
 	}
 
-	&.spectrum-HelpText--negative {
-		.spectrum-HelpText-text {
-			color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
-		}
+	.spectrum-HelpText-validationIcon {
+		color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
+	}
+}
 
-		.spectrum-HelpText-validationIcon {
-			color: var(--highcontrast-helptext-icon-color-default, var(--mod-helptext-icon-color-default, var(--spectrum-helptext-icon-color-default)));
-		}
+.spectrum-HelpText--negative {
+	--spectrum-helptext-content-color-default: var(--spectrum-negative-color-900);
+	--spectrum-helptext-icon-color-default: var(--spectrum-negative-color-900);
```

Comment:
Is there a reason these are using `--spectrum-negative-color-900` instead of `--spectrum-negative-content-color-default`?